packages/mdc-textfield/index.js

```diff
@@ -494,9 +502,10 @@ class MDCTextField extends MDCComponent {
    */
   getFoundationMap_() {
     return {
-      helperText: this.helperText_ ? this.helperText_.foundation : undefined,
-      leadingIcon: this.leadingIcon_ ? this.leadingIcon_.foundation : undefined,
-      trailingIcon: this.trailingIcon_ ? this.trailingIcon_.foundation : undefined,
+      helperText: this.helperText_ ? this.helperText_.foundation_ : undefined,
```

Comment:
the helper-text and icon sub-components have public `foundation` getters so this was correct as-is.